src/pystatis/cube.py

```diff
@@ -105,9 +105,8 @@ def rename_axes(
     Args:
         cube (dict): A dictionary holding the cube data as returned by `parse_cube()`.
         rename_classifying_variables (bool, optional): If True, rename classifying variables.
-            Defaults to True.
-        rename_time_variable (bool, optional): If True, rename the time variable.
-            Defaults to True.
+        Defaults to True.
```

Comment:
not sure about this one but I think Google stylesheet says to intend the line if the string belongs to the previous line so Defaults to True is part of the parameter `rename_classifying_variables`. But not important and if it works with Sphinx fine

Reply:
Another alternative would be to have it (`Defaults...`) all in one line. At least locally this solution complies with black & pylint. The problem with sphinx is, that the indentations (without clear, return-separated blocks) are "unexpected" (warning, no error).